src/Schema/Events/Tap.ts

```diff
@@ -179,3 +179,30 @@ export interface TappedEntityGroup {
 
 export type EntityModuleHeight = "single" | "double"
 export type EntityModuleType = "thumbnail" | "header" | "stub"
+
+/**
+ * A user clicks a Consign button in iOS
+ *
+ * This schema describes events sent to Segment from [[tappedConsign]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedConsign",
+ *    context_screen_owner_type: "sell",
+ *    destination_screen_owner_type: "consignmentSubmission",
+ *    subject: "startSelling"
+ *  }
+ * ```
+ */
+export interface TappedConsign {
+  action: ActionType.tappedConsign
+  context_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_type: ScreenOwnerType
+  subject: Subject
```

Comment:
If the subject is derived from the button text, I think it would be preferable to use an untyped `string` instead of a manually defined type. This way we can pass the button text directly from the UI to segment, which will remove steps in case a button's text is changed.

Reply:
+1 to this suggestion, especially since the designs I'm looking at only have a "start selling" button, but not a "start today" button.

Reply:
Also - since the buttons at the top and bottom of the designs both say "start selling", do we need some other field to identify which of the two buttons they clicked on?

Reply:
- Good point will put a string type
- @pepopowitz: agreed. We can use a context module.
What are the modules called in the code? I am thinking for the one at the top of the page "sellHeader" but I am unsure about the one at the bottom. Is there any name in the code for that footer header?

Reply:
`sellHeader` and `sellFooter` would be good. We call them `HeaderCTA` and `FooterCTA` in the code but it really doesn't matter - anything to distinguish header from footer is great.

Reply:
@abhitip may have suggestions on `ContextModule`

Reply:
`sellHeader` and `sellFooter` sound good to me!